piezo_web_app/PiezoWebApp/src/services/kubernetes/kubernetes_adapter.py

```diff
@@ -13,6 +13,22 @@ def __init__(self, config):
     def delete_namespaced_custom_object(self, group, version, namespace, plural, name, body):
         return self._custom_connection.delete_namespaced_custom_object(group, version, namespace, plural, name, body)
 
+    # pylint: disable=too-many-arguments
+    def delete_options(self,
+                       api_version=None,
+                       dry_run=None,
+                       grace_period_seconds=None,
+                       kind=None, orphan_dependents=None,
```

Comment:
New line between `kind=None,` and `orphan_dependents=None,`

Reply:
d63cc7f